lib/handleRequest.js

```diff
@@ -59,8 +59,12 @@ function handleRequest (req, res, params, context) {
 function handler (reply) {
   var result = validateSchema(reply.context, reply.request)
   if (result) {
-    reply.code(400).send(result)
-    return
+    if (reply.context.attachValidation === false) {
+      reply.code(400).send(result)
+      return
+    } else {
```

Comment:
The `if` block returns so the `else` part is unnecessary. See [this ESLint rule](https://eslint.org/docs/rules/no-else-return) for a more detailed explanation.